src/terrain/routes/token.clj

```diff
@@ -14,4 +14,36 @@
        :summary "Obtain OAuth Tokens"
        :return AccessTokenResponse
        :description-file "docs/get-token.md"
-       (get-token authorization)))))
+       (oauth/get-token authorization))
+
+     (GET "/cas" [:as {{:strs [authorization]} :headers}]
+       :summary "Obtain OAuth Tokens"
+       :return AccessTokenResponse
+       :description-file "docs/get-token.md"
+       (oauth/get-cas-token authorization))
+
+     (GET "/keycloak" [:as {{:strs [authorization]} :headers}]
+       :summary "Obtain Keycloak OIDC Tokens"
+       :return AccessTokenResponse
+       :description-file "docs/get-token.md"
+       (oauth/get-keycloak-token authorization)))))
```

Comment:
For the time being, I decided to have a default `token` endpoint as well as individual endpoints for each identity provider that we support. This allows the use case that users are familiar with to keep working while allowing us to obtain access tokens for specific identity providers. The endpoints for individual identity providers will likely go away when we fully migrate to Keycloak.